README.adoc

```diff
@@ -0,0 +1,78 @@
+    <source width="400" media="(prefers-color-scheme: dark)" srcset="https://www.jenkins.io/images/jenkins-logo-title-dark.svg">
+    <img width="400" src="https://www.jenkins.io/images/jenkins-logo-title.svg">
+  </picture>
+</a>
++++
+
+== About
+
+https://www.jenkins.io/changelog[image:https://img.shields.io/endpoint?url=https%3A%2F%2Fwww.jenkins.io%2Fchangelog%2Fbadge.json[Jenkins Regular Release]]
+https://www.jenkins.io/changelog-stable[image:https://img.shields.io/endpoint?url=https%3A%2F%2Fwww.jenkins.io%2Fchangelog-stable%2Fbadge.json[Jenkins LTS Release]]
+https://hub.docker.com/r/jenkins/jenkins/[image:https://img.shields.io/docker/pulls/jenkins/jenkins.svg[Docker Pulls]]
+https://bestpractices.coreinfrastructure.org/projects/3538[image:https://bestpractices.coreinfrastructure.org/projects/3538/badge[CII Best Practices]]
+
+In a nutshell, Jenkins is the leading open-source automation server.
+Built with Java, it provides over 1,800 https://plugins.jenkins.io/[plugins] to support automating virtually anything,
+so that humans can spend their time doing things machines cannot.
+
+== What to Use Jenkins for and When to Use It
+
+Use Jenkins to automate your development workflow, so you can focus on work that matters most. Jenkins is commonly used for:
+
+* Building projects
+* Running tests to detect bugs and other issues as soon as they are
+introduced
+* Static code analysis
+* Deployment
+
+Execute repetitive tasks, save time, and optimize your development process with Jenkins.
+
+== Downloads
+
+The Jenkins project provides official distributions as WAR files, Docker images, native packages and installers for platforms including several Linux distributions and Windows.
+See the https://www.jenkins.io/download[Downloads] page for references.
+
+For all distributions Jenkins offers two release lines:
+
+* https://www.jenkins.io/download/weekly/[Weekly] -
+  Frequent releases which include all new features, improvements, and bug fixes.
+* https://www.jenkins.io/download/lts/[Long-Term Support (LTS)] -
+  Older release line which gets periodically updated via bug fix backports.
+
+Latest releases:
+https://www.jenkins.io/changelog[image:https://img.shields.io/endpoint?url=https%3A%2F%2Fwww.jenkins.io%2Fchangelog%2Fbadge.json[Jenkins Regular Release]]
+https://www.jenkins.io/changelog-stable[image:https://img.shields.io/endpoint?url=https%3A%2F%2Fwww.jenkins.io%2Fchangelog-stable%2Fbadge.json[Jenkins LTS Release]]
+
+== Source
+
+Our latest and greatest source of Jenkins can be found on https://github.com/jenkinsci/jenkins[GitHub]. Fork us!
+
+== Contributing to Jenkins
+
+Follow the link:CONTRIBUTING.md[contributing guidelines] if you want to propose a change in the Jenkins core.
+For more information about participating in the community and contributing to the Jenkins project,
+see https://www.jenkins.io/participate/[this page].
+
+Documentation for Jenkins core maintainers is in the link:docs/MAINTAINERS.adoc[maintainers guidelines].
```

Comment:
```suggestion
Documentation for Jenkins core maintainers is in the link:MAINTAINERS.adoc[maintainers guidelines].
```
(In case https://github.com/jenkinsci/jenkins/pull/7528 is merged first)